korman/exporter/etlight.py

```diff
@@ -375,6 +375,8 @@ def _prep_for_lightmap(self, bo, toggle):
         # this tended to create sharp edges. There was already a discussion about this on the
         # Guild of Writers forum, so I'm implementing a code version of dendwaler's process,
         # as detailed here: https://forum.guildofwriters.org/viewtopic.php?p=62572#p62572
+        # This has been amended with Sirius's observations in GH-265 about forced uv map
+        # packing. Namely, don't do it unless modifiers make us.
         uv_base = self._get_lightmap_uvtex(mesh, modifier)
```

Comment:
`_get_lightmap_uvtex` appears to still use any available UV channel for the lightmap. Unless the user specifies the UV to use in the lightmap modifier, this method should return nothing to let Korman generate the UVs by itself:
```
def _get_lightmap_uvtex(self, mesh, modifier):
    if modifier.uv_map:
        return mesh.uv_textures[modifier.uv_map]
    return None
```